src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2426,10 +2434,98 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-lower"
+    title={<InlineCode>lower(str)</InlineCode>}
+  >
+    Use the lower() function transliterates all alphabetic characters of a string value to lower-case.
+
+    Arguments:
+    * str - The string value to be transliterated to lower-case
+      * This can be anything that evaluates to a string, including a literal string in quotes, a queried string attribute, a function that returns a string, or even a subquery that returns a single string value.
+      * If this argument evaluates to null, the lower function will return null.
```

Comment:
All instances of function names like "lower" should have parens and backticks, for consistency with other entries (e.g. `aggregationendtime()`)